python/lsst/ts/ess/csc/rpi_data_client.py

```diff
@@ -665,3 +830,72 @@ async def process_telemetry(
             self.log.warning(
                 f"Ignoring telemetry for sensor {sensor_name} with unknown response code {response_code}"
             )
+
+    async def write_humidity_etc(
```

Comment:
Maybe consider renaming this to `write_humidity_and_related_quantities`?

Reply:
I'm not keen on this because I don't think it adds much information.

Reply:
OK, it was merely a suggestion.